sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -1931,7 +2017,7 @@ public virtual Response DeleteEventRoute(string eventRouteId, CancellationToken
             string digitalTwinId,
             string messageId,
             string payload,
-            PublishTelemetryOptions options = null,
+            DateTimeOffset? timeStamp = null,
```

Comment:
I wonder if this should be cased `timestamp`. The only other case of timestamp I found in .NET is `Stopwatch.GetTimestamp`.

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.gettimestamp?view=netcore-3.1

Reply:
Krzysztof wrote `Stopwatch`.  I'll probably get yelled at if we don't follow it.  :smile: